code/drasil-data/lib/Data/Drasil/Quantities/Physics.hs

```diff
@@ -44,7 +44,8 @@ acceleration, angularAccel, angularDisplacement, angularVelocity, chgInVelocity,
   yVel, momentum, moment, moment2D, fOfGravity, positionVec, tension, angularFrequency, 
   period, frequency, chgMomentum :: UnitalChunk
 
-acceleration           = uc CP.acceleration           (Concat [vec lA, label "(", lT, label ")"]) (Vect Real) accelU
+-- TODO: what should the type here be?
+acceleration           = uc CP.acceleration           (Concat [vec lA, label "(", lT, label ")"]) undefined   accelU
```

Comment:
Taking a look at usage in GamePhysics, a `Real -> Real` function seems more likely than a `Vect Real`.

Reply:
Note that functions are still mostly unsupported as well, hence why the time variable is manually added to the symbol of acceleration.

Reply:
Another option for the type is `Real -> Vect Real`.  In 3D the acceleration is a function of time.  In general, at any time, we can have 3 components of acceleration.  We also have a 1D version of acceleration in Game Physics which would have the type `Real -> Real`.

Reply:
This is a symptom of a deeper problem in Drasil: acceleration should exist in many different ways (as a concept, as an instance of properties for several objects, but also as something that may be represented in different ways!)

So `acceleration` can be `Real`, `Vect Real` and `Real -> Real` depending on the case at hand. Thus we need to be able to do that.

Reply:
So this is where @CSchank is likely going to need to rely on @balacij sorting out some stuff in our chunks.

For this one: I don't think the *concept* of `acceleration` should be a `UnitalChunk` because its 'units' will actually depend on the setting in which it is used, i.e. acceleration gets a unit once the object of concern is known.

We could treat it as a vector of unknown dimension if we insisted on giving it units 'early', thus indicating that it is a *dependent* concept.

Reply:
@balacij any chance we can work on this?

Reply:
@balacij might now be free, given that his Comp II was earlier today.

Reply:
Hi, sorry @CSchank! I will message you on Discord!